src/resmom/mom_main.c

```diff
@@ -3980,8 +3979,8 @@ set_vnode_per_numa_node(char *value)
  *	Set the alps_release_wait_time in micoseconds to wait between ALPS release
  *	reservation requests
  *
- * @retval 0 HANDLER_FAIL
- * @retval 1 HANDLER_SUCCESS
+ * @retval HANDLER_FAIL
```

Comment:
Usually the retval statements say what you are returning and when it will be returned.  In your case, it's the return value of set_float().  Maybe replace these two with "@return return value of set_float()" or something like that.